doc: known issues: add Bluetooth LE entry (#17684)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

  **Workaround:** Either disable host flow control (:kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL`) or cherry-pick commits from the upstream: `Zephyr PR #65272 <https://github.com/zephyrproject-rtos/zephyr/pull/65272>`_.

> DRGN-23511: Building of multilink Bluetooth applications fails if :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` is not set explicitly.

Reviewer: ```suggestion
.. rst-class:: v2-7-0

DRGN-23511: Building of multilink Bluetooth applications fails
  This happens when the :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` Kconfig option is not explicitly set.
```
Add any other affected versions to the first line.

>   **Workaround:** Increase the Kconfig value accordingly.

Reviewer: ```suggestion
  **Workaround:** Set the Kconfig value explicitly.
```

> DRGN-23518: Reassembly broken when :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` is too small.

Reviewer: ```suggestion
DRGN-23518: Reassembly is sometimes broken
  This happens when the :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` Kconfig option is set to a value that is too small.
```

  **Workaround:** Either disable host flow control (:kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL`) or cherry-pick commits from the upstream: `Zephyr PR #65272 <https://github.com/zephyrproject-rtos/zephyr/pull/65272>`_.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1

DRGN-23518: Reassembly is sometimes broken

>   This happens when the :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` Kconfig option is set to a value that is too small.

Reviewer 2: what means "too small", can you rephrase it in a quantifiable description?

  **Workaround:** Either disable host flow control (:kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL`) or cherry-pick commits from the upstream: `Zephyr PR #65272 <https://github.com/zephyrproject-rtos/zephyr/pull/65272>`_.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1

> DRGN-23518: ACL reassembly may deadlock if `CONFIG_BT_BUF_ACL_RX_COUNT <= CONFIG_BT_MAX_CONN`.

Reviewer: ```suggestion
DRGN-23518: ACL reassembly may deadlock
  This can happen if  the value of :kconfig:option:`CONFIG_BT_BUF_ACL_RX_COUNT` Kconfig option is less than or equal to the value of :kconfig:option:`CONFIG_BT_MAX_CONN`.
```

>   **Workaround:** Ensure `CONFIG_BT_BUF_ACL_RX_COUNT > CONFIG_BT_MAX_CONN`.

Reviewer: ```suggestion
  **Workaround:** Ensure that the value of :kconfig:option:`CONFIG_BT_BUF_ACL_RX_COUNT` is greater than the value of :kconfig:option:`CONFIG_BT_MAX_CONN`.
```
